README.md

```diff
@@ -129,28 +186,59 @@ Once you are finished with the reference architecture, you can remove all provis
    terraform destroy
    ```
 
-## Terraform docs
+### Terraform docs
 
 <!-- BEGIN_TF_DOCS -->
-### Requirements
+#### Requirements
```

Comment:
Those bits are autogenerated, so you would need to adjust https://github.com/humanitec-architecture/reference-architecture-aws/blob/main/docs/.terraform-docs.yaml#L13 (change 3 ~> 4) to make this change.

Just keep in mind that this config is also used to generate https://github.com/humanitec-architecture/reference-architecture-aws/blob/main/examples/with-backstage/README.md , so we either need two configs or also adjust the other readme.

Let me know if you need help :)

Reply:
Thanks for the explanation. Fxied, please check.